eth-contracts/migrations/utils.js

```diff
@@ -0,0 +1,3 @@
+module.exports.logBytecodes = (contract) => {
+  console.log(`${contract.contractName} || bytecode: ${contract.bytecode.length} || deployedBytecode: ${contract.deployedBytecode.length}`)
```

Comment:
this is used in a single migration, does it really need to be  in a separate file? let's remove this file altogether and use the function in the file where it's required